ui-tests/cypress/integration/components/ux_fragments.spec.js

```diff
@@ -1,808 +1,665 @@
-              page.getContent().getEditBtn().should('exist').and('be.visible');
-              page.getContent().getReferencedUxFragments().should('exist').and('be.visible').and('contain.text', 'Referenced UX fragments');
-              page.getContent().getReferencedPageTemplates().should('exist').and('be.visible').and('contain.text', 'Referenced page templates');
-              page.getContent().getReferencedWidgetTypes().should('exist').and('be.visible').and('contain.text', 'Referenced widget types');
-            });
-      });
-    });
 
-  });
-  describe('Sanity Tests', () => {
-    describe('Actions', () => {
-      it([Tag.SANITY, 'ENG-3522'], 'Searching a fragment', () => {
-        addRandomFragment().then(fragment => {
-          openFragmentsPage()
-              .then(page =>
-                  page.getContent().getSearchCodeInput().then(input => page.getContent().type(input, fragment.code)))
-              .then(page => page.getContent().clickSearchSubmitButton())
-              .then(page =>
-                  page.getContent().getTableRows().should('have.length', 1)
-                      .each(row => cy.get(row).children(htmlElements.td).eq(0).should('contain', fragment.code)));
-        });
+    describe('Existing fragment', () => {
+
+      before(() => {
+        cy.kcAPILogin();
+        generateRandomFragment().then(fragment =>
+            cy.fragmentsController()
+              .then(controller => controller.addFragment(fragment))
+              .then(response => Cypress.env('fragmentToBeDeleted', response.body.payload)));
+      });
+
+      after(() => {
+        cy.kcAPILogin();
+        cy.wrap(Cypress.env('fragmentToBeDeleted')).then(fragmentToBeDeleted =>
+            cy.fragmentsController().then(controller => controller.deleteFragment(fragmentToBeDeleted.code)));
+      });
+
+      it([Tag.SMOKE, 'ENG-3522'], 'Actions dropdown is displayed', () => {
+        cy.wrap(Cypress.env('fragmentToBeDeleted')).then(fragment =>
+            openUXFragmentsPage()
+                .then(page => {
+                  const kebabMenu = page.getContent().getKebabMenu(fragment.code).open();
+                  kebabMenu.getDropdown().should('be.visible');
+                  kebabMenu.getEdit()
+                           .should('be.visible')
+                           .should('have.text', `Edit ${fragment.code}`);
+                  kebabMenu.getClone()
+                           .should('be.visible')
+                           .should('have.text', `Clone ${fragment.code}`);
+                  kebabMenu.getDetails()
+                           .should('be.visible')
+                           .should('have.text', `Details for: ${fragment.code}`);
+                  kebabMenu.getDelete()
+                           .should('be.visible')
+                           .should('have.text', 'Delete');
+                }));
+      });
+
+      it([Tag.SMOKE, Tag.FEATURE, 'ENG-3522'], 'Edit page is displayed', () => {
```

Comment:
Use a single test, checking which tag is being executed, when the scenario differs only on validations